main.py

```diff
@@ -1,10 +1,43 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("db/tests/players.json", "r") as file:
+        players = json.load(file)
+
+    for nickname, details in players.items():
+        email = details["email"]
+        bio = details["bio"]
+        race = details["race"]
+        guild = details["guild"]
+
+        race_obj, _ = Race.objects.get_or_create(
+            name=race["name"],
+            description=race["description"],
+        )
+
+        for skill in race["skills"]:
+            Skill.objects.get_or_create(
+                name=skill["name"],
+                bonus=skill["bonus"],
+                defaults={"race": race_obj},
+            )
+
+        guild_obj = Guild.objects.get_or_create(
+            name=guild["name"],
+            description=guild["description"],
+        )[0] if guild else None
```

Comment:
```suggestion
        ) if guild else None
```

Reply:
@vasylhnatiuk There is a piece of code after the brackets "if guild else None". If I replace "[0]" with "_", I will have to use something like "if guild else (None, None)" which looks worse in my opinion.

Reply:
you use the same ` if guild else None`